docs/getting-started.md

```diff
@@ -30,7 +30,9 @@ with the following flags:
 flag.
 
 As for OVS, when using the built-in kernel module, kernel version >= 4.6 is
-required. On the other hand, when building it from OVS sources, OVS
+required, please be aware that `vport-stt` module is not shipped in built-in kernel module,
+please load it manually before STT tunnel is enabled.
```

Comment:
STT is rarely used. Better not to add its requirement here to avoid bothering most users. Maybe append it after "In case a node does not have a supported OVS module installed, you can install it following the instructions at: [Installing Open vSwitch]".